llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -207,6 +207,14 @@ Example 7 - Addresses as symbol names:
   foz
   /tmp/test.h:1:0
 
+Example 8 - Skip line zero output for an address with no line correspondence (an address associated with line zero):
```

Comment:
Sorry - I know I suggested it but looking at it again "Skip line zero output" reads as "Skip (line zero output)" at least to me, which is confusing. Let's change to "--skip-line-zero output".

Reply:
Just to be clear do you want in this fashion?

``` 
--skip-line-zero output for an address with no line correspondence (an address associated with line zero)
```

Or 

```
Skip line-zero output for an address with no line correspondence (an address associated with line zero)
```

Reply:
A quick glance, and I'd prefer the first one (and I'm pretty sure @bd1976bris meant this too).

As this is documentation, please make sure to use the appropriate `:option:` markup to link the `--skip-line-zero` reference to the actual otpion.

Reply:
Done